app/styles/app/modules/request/form.scss

```diff
@@ -0,0 +1,116 @@
+.request-configs-form {
+  margin-top: 2em;
+}
+
+.request-configs-form .travis-form {
+  .config-type {
+    position: absolute;
+    top: 1px;
+    right: 0;
+    font-size: 10px;
+    z-index: 2;
+
+    .javascript,
+    .yaml {
+      display: inline-block;
+      padding: 0 8px;
+      border: 1px solid transparent;
+      border-radius: 4px;
+    }
+  }
+
+  .javascript .javascript,
+  .yaml .yaml {
+    border-color: lighten($cement-grey, 15);
+  }
+  .yaml .ember-tooltip {
+      left: -20px !important;
+  }
+  .javascript .ember-tooltip {
+      left: 20px !important;
+  }
+
+  .travis-form__field .travis-form__field-component {
+    box-shadow: inset 0 0 1px 1px $dry-cement;
+  }
+
+  .travis-form__field .travis-form__field-checkbox {
+    box-shadow: none;
+  }
+
+  .request-configs-fields-container {
+    display: flex;
+    flex-flow: row wrap;
+
+    .form-elem {
+      &:first-child {
+        flex-grow: 2;
+        margin-right: 0.5em;
+      }
+
+      &:last-child {
+        flex-grow: 1;
+        margin-left: 0.5em;
+      }
+    }
+  }
+
+  .CodeMirror {
+    height: 350px;
+    margin: -4px;
+    padding: 8px 12px 8px 4px;
+    width: calc(100% + 8px);
+    border-radius: 10px;
+    border: 4px solid transparent;
+    box-shadow: inset 0 0 1px 1px $dry-cement;
+    transition: border-color 0.2s ease-in;
+    color: $asphalt-grey;
+    font-size: 12px;
+    font-family: Cousine, monospace;
+    line-height: 1.7;
+
+    &.CodeMirror-focused {
+      outline: none;
+      border-color: rgba(62, 170, 175, 0.2);
+    }
+
+    pre {
+      padding: 0 14px;
+    }
+  }
+
+  .CodeMirror-placeholder {
+    color: $cement-grey;
+  }
+
+  .CodeMirror-gutters {
+    border: 0;
+    background-color: transparent;
+  }
+
+  h4,
+  label.ref-type {
+    margin-bottom: 1em;
+  }
+
+  label.ref-type {
+    display: inline-block;
+  }
+
+  input[type=radio] {
+    margin-right: 6px;
+  }
+
+  .ember-radio-button {
+    margin-right: 16px;
+  }
```

Comment:
As long as we don't use `ember-radio-button`, would it make sense to remove this style?

Reply:
true, probably a left over from before the radio button PR. removed:

Reply:
Thanks 👍